sqlmesh/core/model/meta.py

```diff
@@ -212,6 +214,34 @@ def _columns_validator(
 
         return v
 
+    @field_validator("column_descriptions_", mode="before")
+    @field_validator_v1_args
+    def _column_descriptions_validator(
+        cls, vs: t.Any, values: t.Dict[str, t.Any]
+    ) -> t.Optional[t.Dict[str, str]]:
+        if vs is None:
+            return None
+
+        if isinstance(vs, exp.Paren):
+            vs = vs.flatten()
+
+        if isinstance(vs, (exp.Tuple, exp.Array)):
+            vs = vs.expressions
+
+        col_descriptions = (
+            vs if isinstance(vs, dict) else {v.this.name: v.expression.alias_or_name for v in vs}
```

Comment:
I believe we can _slightly_ simplify this; there should be no alias in the right-hand side, so `alias_or_name` doesn't seem suitable:
```suggestion
            vs if isinstance(vs, dict) else {v.name: v.expression.name for v in vs}
```